ports/ports.go

```diff
@@ -52,6 +54,14 @@ func PortToHostPort(port int) string {
 	return ":" + strconv.Itoa(port)
 }
 
+// HostPortToPort converts the  host:port address string intoto port int
+func HostPortToPort(hostPort string) (int, error) {
```

Comment:
Could [`netutils.GetPort()`](https://github.com/jaegertracing/jaeger/blob/master/pkg/netutils/port.go#L23) be used instead?

Reply:
Thanks.  This method lead to the `SplitHostPort()` method in the `net` package.